### DISCUSSION TITLE
Disabling testing when executing 'make'

### DISCUSSION THREAD

**Discussion by omerbn** [taocpp/taopq#20] [Q&A] [answered]
How do I do it? :)

**Comment by ColinH** [comment, marked as answer]
Have you tried `make compile` instead of `make`?

**Comment by omerbn** [comment]
I haven't. i'm noob in `make` procedure (working c/c++ on linux)   :)

So do iinclude the files into my code?
after running `make compile`, should I copy `build/lib/libtaopq.a` to `/usr/local/lib` and `build/src/lib/pq` to `usr/local/include/` ?
